config/data/courts.csv

```diff
@@ -0,0 +1,251 @@
+C,Harrow Crown Court
+B,Hastings Magistrates' Court
+B,Hatfield Magistrates' Court
+B,Haverfordwest Magistrates' Court
+B,Hendon Magistrates' Court
+C,Hereford Crown Court
+B,Herefordshire Magistrates' Court
+B,Highbury Corner Magistrates' Court
+B,Horsham Magistrates' Court
+B,Huddersfield Magistrates' Court
+B,Hull Magistrates' Court
+B,Huntingdon Magistrates' Court
+C,Inner London Crown Court
+C,Ipswich Crown Court
+B,Ipswich Magistrates' Court
+B,Isle of Wight Magistrates' Court
+C,Isleworth Crown Court
+B,Kidderminster Magistrates' Court
+C,King's Lynn Crown Court
+C,Kingston upon Thames Crown Court
+C,Kingston-upon-Hull Crown Court
+C,Lancaster Crown Court
+B,Lancaster Magistrates' Court
+B,Lavender Hill Magistrates' Court
+B,Leamington Spa Magistrates' Court
+C,Leeds Crown Court
+B,Leeds Magistrates' Court
+C,Leicester Crown Court
+B,Leicester Magistrates' Court
+C,Lewes Crown Court
+C,Lincoln Crown Court
+B,Lincoln Magistrates' Court
+C,Liverpool Crown Court
+B,Liverpool Magistrates' Court
+B,Llandrindod Wells Magistrates' Court
+B,Llandudno Magistrates' Court
+B,Llanelli Magistrates' Court
+B,Loughborough Magistrates' Court
+C,Luton Crown Court
+B,Luton Magistrates' Court
+B,Maidenhead Courthouse
+C,Maidstone Crown Court
+B,Maidstone Magistrates' Court
+B,Manchester City Magistrates' Court
+C,Manchester Crown Court
+B,Mansfield Magistrates' Court
+B,Margate Magistrates' Court
+B,Medway Magistrates' Court
+C,Merthyr Tydfil Crown Court
+B,Merthyr Tydfil Magistrates' Court
+B,Milton Keynes Magistrates' Court
+C,Minshull Street Manchester Crown Court
+C,Mold Crown Court
+B,Mold Magistrates' Court
+B,Newcastle Magistrates' Court
+C,Newcastle upon Tyne Crown Court
+B,Newcastle upon Tyne Magistrates' Court
+C,Newport (I.O.W.) Crown Court
+C,Newport (South Wales) Crown Court
+B,Newport Magistrates' Court
+B,Newport Magistrates' Court (Crown Court Bldg)
+B,Newton Abbot Magistrates' Court
+B,Newton Aycliffe Magistrates' Court
+B,North Shields Magistrates' Court
+B,North Somerset Magistrates' Court
+B,North Staffordshire Justice Centre
+B,Northallerton Magistrates' Court
+C,Northampton Crown Court
+B,Northampton Magistrates' Court
+C,Norwich Crown Court
+B,Norwich Magistrates' Court
+C,Nottingham Crown Court
+B,Nottingham Magistrates' Court
+B,Nuneaton Magistrates' Court
+C,Oxford Crown Court
+B,Oxford Magistrates' Court
+C,Peterborough Crown Court
+B,Peterborough Magistrates' Court
+B,Peterlee Magistrates' Court
+C,Plymouth Crown Court
+B,Plymouth Magistrates' Court
+B,Poole Magistrates' Court
+C,Portsmouth Crown Court
+B,Portsmouth Magistrates' Court
+C,Preston Crown Court
+B,Preston Magistrates' Court
+C,Reading Crown Court
+B,Reading Magistrates' Court
+B,Redditch Magistrates' Court
+B,Romford Magistrates' Court
+C,Salisbury Crown Court
+B,Salisbury Magistrates' Court
+B,Scarborough Magistrates' Court
+B,Sevenoaks Magistrates' Court
+C,Sheffield Crown Court
+B,Sheffield Magistrates' Court
+C,Shrewsbury Crown Court
+C,Shrewsbury Crown Court

```

Comment:
Seems to be a double entry here as well

Reply:
Good eye. This is definitely a duplicate. Will remove it and update the digest.